src/components/button-toggle-group/button-toggle-group.stories.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, boolean } from '@storybook/addon-knobs';
+import notes from './notes.md';
+import ButtonToggle from '../button-toggle/button-toggle';
+import ButtonToggleGroup from './button-toggle-group';
+
+storiesOf('Button Toggle Group', module)
+  .addParameters({
+    info: {
+      inline: true,
+      header: false,
+      propTablesExclude: [ButtonToggle],
+      propTables: [ButtonToggleGroup],
+      source: false
+    }
+  }).add('default', () => {
+    const timeToDisappear = text('timeToDisappear', '');
```

Comment:
You could consider refactoring this so that fewer lines are taken up declaring and then assigning these props on the component.

Reply:
It's for readability, I suggest to keep it.